mom/Graphite.py

```diff
@@ -0,0 +1,59 @@
+# Memory Overcommitment Manager
+# Copyright (C) 2010 Adam Litke, IBM Corporation
+#
+# This program is free software; you can redistribute it and/or modify
+# it under the terms of the GNU General Public License version 2 as
+# published by the Free Software Foundation.
+#
+# This program is distributed in the hope that it will be useful, but
+# WITHOUT ANY WARRANTY; without even the implied warranty of
+# MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE. See the GNU
+# General Public License for more details.
+#
+# You should have received a copy of the GNU General Public
+# License along with this program; if not, write to the Free Software
+# Foundation, Inc., 51 Franklin St, Fifth Floor, Boston, MA 02110-1301 USA
+
+import time
+import ConfigParser
+import socket
+import logging
+
+class Graphite:
+    def __init__(self, graphite_host, graphite_port, graphite_protocol, name):
+    
+      self.graphite_host = graphite_host
+      self.graphite_port = graphite_port
+      self.graphite_protocol = graphite_protocol
+      self.vmname = name
```

Comment:
`vmname` is misleading. It could also be `HostMonitor` for host monitor. Maybe better to use something like `monitor_name`.